.github/workflows/smoke.yml

```
@@ -5,10 +5,24 @@ on:
   pull_request_target:
     paths: 'task/**'
   schedule:
-  - cron: '0 9 * * *'
+  - cron: '0 1 * * *' # Everyday @ 0100 UTC
+  workflow_dispatch:
+    inputs:
+      test_target:
+        description: Type of instances to be provisioned during test.
+        required: true
+        default: 'gpu+spot'
+        type: choice
+        options:
+          - quick
+          - quick+spot
+          - gpu
+          - gpu+spot
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref_name }}
   cancel-in-progress: true
+env:
+ SMOKE_TEST_TARGET: ${{ github.event_name == 'pull_request_target' && 'quick' || inputs.test_target }}
```

```suggestion
 SMOKE_TEST_TARGET: ${{ github.event_name != 'pull_request_target' && 'quick' || inputs.test_target || 'gpu+spot' }}
```

Matbe make `gpu+spot` the default for the `on: schedule` daily tests?